src/main/java/net/wurstclient/serverfinder/CleanUpScreen.java

```diff
@@ -125,6 +125,7 @@ private void cleanUp()
 		{
 			ServerEntry server = prevScreen.getServerList().get(i);
 			if(cleanupUnknown
+				&& server.label != null
 				&& server.label.equals("\u00a74Can\'t resolve hostname")

```

Comment:
I would prefer to do it the other way around:
```java
"\u00a74Can\'t resolve hostname".equals(server.label)
```
This fixes the issue in just one line.